bin/check_fixmystreet

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env perl
+
+use strict;
+use lib '/usr/lib/nagios/plugins';
+use utils qw(%ERRORS);
+use Getopt::Long;
+
+BEGIN {
+    use File::Basename qw(dirname);
+    use File::Spec;
+    my $d = dirname(File::Spec->rel2abs($0));
+    require "$d/../../fixmystreet/setenv.pl";
+}
+
+my $reports = 50;
```

Comment:
Now the daemon is running, plus this query will ignore Noop-set bodies, this could probably be much lower, I'd hope.

Reply:
I've made it 10 for now. In theory it is configurable, although currently the automation for the monitoring scripts doesn't support arguments, I think. It would be simple enough to add though.